src/generators/web.cr

```diff
@@ -10,8 +10,13 @@ class LuckyCli::Generators::Web
     project_name : String,
     @options : Options
   )
-    @project_dir = project_name.gsub(" ", "_")
-    @project_name = @project_dir.gsub("-", "_")
+    @project_dir = project_name
+    @project_name = project_name
+
+    validate(directory_name: @project_dir)
+    validate(project_name: @project_name)
```

Comment:
What if instead of validating these separately, just the project name is validated. Since a valid project name will always be a valid directory name, that should cut down on the number of tests and the number of supporting lines of code